src/components/Popups/RedirectPopup.js

```diff
@@ -8,11 +8,16 @@ import Spinner from '../../components/Spinner';
 
 const RedirectPopup = ({ loading, availableCredentialConfigurations, handleClose, handleContinue, popupTitle, popupMessage }) => {
 	const { t } = useTranslation();
-	const [selectedConfiguration, setSelectedConfiguration] = useState(null);
+	const [selectedConfiguration, setSelectedConfiguration] = useState('pid-sd-jwt');
+
+	const credentialConfigurationIdDisplayGlossary = {
+		"pid-sd-jwt": "SD JWT PID",
+		"pid-mso-mdoc": "mDoc PID"
```

Comment:
Should these be translated?
```suggestion
		"pid-sd-jwt": t("SD JWT PID"),
		"pid-mso-mdoc": t("mDoc PID"),
```